### ISSUE TITLE
Sanity check for refnum attributes at StepCounter

### ISSUE THREAD

**Issue by dginev** [brucemiller/LaTeXML#116]
[Originally Ticket 1440]

Currently, the StepCounter subroutine at Package.pm is happy with any expanded list of tokens and passes them on to the refnum attributes. 

However, this can lead to atomic (presentational?) command sequences being propagated all the way to the labels at the XHTML files, as in [this example](http://arxmliv.kwarc.info/files/math-ph/papers/0009016/0009016.xhtml) (look at the section numbers).

Currently, the StepCounter implementation looks like:

```
sub StepCounter {
  my($ctr)=@_;
  my $value = CounterValue($ctr);
  AssignValue("\\c\@$ctr"=>$value->add(Number(1)),'global');
  DefMacroI(T_CS("\\\@$ctr\@ID"),undef, Tokens(Explode(LookupValue('\c@'.$ctr)->valueOf)),
        scope=>'global');
  # and reset any within counters!
  if(my $nested = LookupValue("\\cl\@$ctr")){
    foreach my $c ($nested->unlist){
      ResetCounter($c->toString); }}
  Expand(T_CS("\\the$ctr")); }
```

I propose to improve the final line to:

```
 ...
 my $expanded = Expand(T_CS("\\the$ctr"));
  @$expanded = grep($_->getCatcode != CC_CS,@$expanded);
  $expanded; }
```

In other words, to strip away any remaining command sequences which would be propagated to the refnum attribute. This of course loses information (in the example, we won't remember that we are supposed to have a large bold section number), but since refnum is an attribute we can't do much better. Ideally, we should have a <refnum> element for which we could provide more details.

How does such a change to StepCounter sound to you?
